lib/octokit/client/repositories.rb

```diff
@@ -41,11 +42,13 @@ def repository(repo, options = {})
       # @option options [String] :private `true` makes the repository private, and `false` makes it public.
       # @option options [String] :has_issues `true` enables issues for this repo, `false` disables issues.
       # @option options [String] :has_wiki `true` enables wiki for this repo, `false` disables wiki.
+      # @option options [Boolean] :is_template `true` makes the repository a template, `false` makes it not a template.
       # @option options [String] :has_downloads `true` enables downloads for this repo, `false` disables downloads.
       # @option options [String] :default_branch Update the default branch for this repository.
       # @return [Sawyer::Resource] Repository information
       def edit_repository(repo, options = {})
         repo = Repository.new(repo)
+        options = ensure_api_media_type(:template_repositories, options) if options.include? :is_template
```

Comment:
I'm usually a fan of guard clauses, but this feels a bit terse.

Can we update it?

```ruby
if options.include? :is_template
  options = ensure_api_media_type(:template_repositories, options)
end
```